fundingmanager.go

```diff
@@ -912,8 +912,20 @@ func (f *fundingManager) stateStep(channel *channeldb.OpenChannel,
 				err)
 		}
 
+		// As the fundingLocked message is now sent to the peer, the
+		// channel is moved to the next state of the state machine. It
+		// will be moved to the last state (actually deleted from the
+		// database) after the channel is finally announced.
+		err = f.saveChannelOpeningState(&channel.FundingOutpoint,
```

Comment:
Folding should be: 
```go
f.saveChannelOpeningState(
     &channel.FundingOutpoint, fundingLockedSent, shortChanID,
)
```

Reply:
Alternatively, we can have this logic be present in the caller function, then based on the final state returned,  either persist or delete the funding state.

Reply:
Fixed folding: addedToRouterGraph